tests/test_infer/test_inference_engine.py

```diff
@@ -5,15 +5,16 @@
 import torch
 import torch.distributed as dist
 from torch.multiprocessing import Manager
-from transformers import AutoTokenizer, GenerationConfig, LlamaConfig, LlamaForCausalLM
+from transformers import BloomForCausalLM, BloomTokenizerFast, GenerationConfig
```

Comment:
Better prevent modifying this pytest. You might want to test the model locally, or create a pytest in `test_models/` (we'll refactor these tests anyway)